### PR TITLE
[1LP][RFR] moving cleanup_vm def to common.vm from common.provider

### PR DESCRIPTION
**Pull request by izapolsk**
Someone put cleanup_vm def into common.provider instead of common.vm.

### CODE REVIEW

File: cfme/common/vm.py [hunk 1]
```diff
@@ -957,3 +957,12 @@ def __init__(self, name, provider, template_name=None):
...
         return self.provider.mgmt.does_template_exist(self.name)
+
+
+def cleanup_vm(vm_name, provider):
+    try:
+        logger.info('Cleaning up VM %s on provider %s', vm_name, provider.key)
+        provider.mgmt.delete_vm(vm_name)
+    except:
+        # The mgmt_sys classes raise Exception :\
+        logger.warning('Failed to clean up VM %s on provider %s', vm_name, provider.key)
```
Comment (sshveta) [inline cfme/common/vm.py]: Optional: This code can be used in https://github.com/ManageIQ/integration_tests/blob/master/cfme/fixtures/vm_name.py#L14
also .

Comment (izapolsk) [reply cfme/common/vm.py]: thanks but  this is small part of huge task and I don't want to deviate because each change requires much time for testing.

**Comment by bsquizz** [comment]
I already have a PR addressing cleanup VM stuff. #6866

File: cfme/common/vm.py (hunk 1 above)
Comment (sshveta) [reply cfme/common/vm.py]: :+1:

**Comment by izapolsk** [comment]
@bsquizz, thanks, I close my PR then.

**Pull request closed without merging**